dali/test/python/test_operator_numba_func.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+import os
+from numba import cfunc, types, carray
+
+from nvidia.dali import pipeline_def
+import nvidia.dali as dali
+import nvidia.dali.fn as fn
+import nvidia.dali.types as dali_types
+from test_utils import get_dali_extra_path
+
+test_data_root = get_dali_extra_path()
+lmdb_folder = os.path.join(test_data_root, 'db', 'lmdb')
+
+fun1_sig = types.void(types.CPointer(types.uint8),  # Pointer to output sample
```

Comment:
`fun1_sig` sound too generic.
The same for `fun2_sig`, `setup1_sig` and `fun3_sig`.